targets/TMS/main.cpp

```diff
@@ -120,13 +120,18 @@ int main() {
     timer.stopTimer();
 
     // Set up TMS and necessary device drivers
-    TMS::TMS tms;
-    IO::PWM& pwm = IO::getPWM<IO::Pin::PC_0>();
-    TMS::HeatPump pump = TMS::HeatPump(pwm);
-    TMS::RadiatorFan fans[] = {
-        TMS::RadiatorFan(IO::getGPIO<IO::Pin::PC_1>()),
-        TMS::RadiatorFan(IO::getGPIO<IO::Pin::PB_0>())
+    IO::ADC* adcs[4] = {
```

Comment:
This is fine for now, this will need to be changed in the future to make use of the mux since in reality we are only using 1 ADC pin